tm2/Makefile

```diff
@@ -7,18 +7,16 @@ help:
 rundep=go run -modfile ../misc/devdeps/go.mod
 
 .PHONY: build
-build: _build.tm2txsync
 _build.tools: _build.aminoscan _build.goscan _build.logjack _build.iaviewer
 
-_build.tm2txsync:;  go build -o build/tm2txsync  ./cmd/tm2txsync
 _build.aminoscan:;  go build -o build/aminoscan  ./pkg/amino/cmd/aminoscan
 _build.goscan:;     go build -o build/goscan     ./pkg/amino/cmd/goscan
 _build.logjack:;    go build -o build/logjack    ./pkg/autofile/cmd
 _build.iaviewer:;   go build -o build/iaviewer   ./pkg/iavl/cmd/iaviewer
 
 .PHONY: install
 install:
-	go install ./cmd/...
+#	go install ./cmd/...
```

Comment:
maybe remove entire install command?

Reply:
It will make sense soon, we can comment now and reactivate later, or just let as it is if it's not generating errors/warnings.